src/test/ui/consts/miri_unleashed/mutable_references.rs

```diff
@@ -1,20 +1,18 @@
 // compile-flags: -Zunleash-the-miri-inside-of-you
+#![feature(const_mut_refs)]
 #![allow(const_err)]
 
 use std::cell::UnsafeCell;
 
 // a test demonstrating what things we could allow with a smarter const qualification
 
 static FOO: &&mut u32 = &&mut 42;
-//~^ WARN: skipping const checks
 
 static BAR: &mut () = &mut ();
```

Comment:
Leave a comment that this is fine because accessing an immutable static `BAR` is equivalent to accessing `*&BAR` which puts the mutable reference behind an immutable one